Dockerfile

```diff
@@ -1,22 +1,13 @@
-FROM python:3.7-alpine
+FROM python:3.6
 
-MAINTAINER Indrani Biswas <ibiswas.github.io>
-
-EXPOSE 8000
-
-RUN apk add --no-cache jpeg-dev zlib-dev
-RUN apk add --no-cache --virtual .build-deps build-base linux-headers \
-    && pip install Pillow
-
-ADD . /ecommerce
-
-WORKDIR /ecommerce
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
 
+WORKDIR /usr/src/app
+COPY requirements.txt ./
 RUN pip install -r requirements.txt
+COPY . .
 
-RUN python ecommerce/manage.py makemigrations
```

Comment:
Migrations should be a part of Docker commands

Reply:
Reverted.